env-defaults

```diff
@@ -39,6 +39,9 @@ PORT=3000
 # callbacks
 BACKEND_PORT=3001
 
+# The port on which a worker container listens for work from the manager container
+WORKER_PORT=5001
```

Comment:
Is this used by the main harmony application or this env needed elsewhere? Just wondering if we're putting things in env-defaults that aren't related to the main harmony app and we should split them into a different place.

Reply:
I have this here because this is the file we use when we run `bin/deploy-services`/`bin/restart-services`. Not sure where else I can put env vars that will get picked up by those scripts.